controllers/reviews.controller.js

```diff
@@ -0,0 +1,12 @@
+const {fetchReview} = require("../models/reviews.model.js")
+
+
+exports.getReviews = (req, res, next) => {
+    fetchReview(req.params.id)
+        .then((reviewData)=>{
+            res.status(200).send(reviewData)
```

Comment:
We should be sending data back on a key that is appropriately named (e.g. review), not just the data object itself.